HVZ.Persistence.MongoDB/Repos/GameRepo.cs

```diff
@@ -175,17 +181,79 @@ public async Task<Game> AddPlayer(string gameId, string userId)
         return newGame;
     }
 
-    public async Task<Game> SetActive(string gameId, bool active, string updatorId)
+    public async Task<Game> StartGame(string gameId, string instigatorId)
     {
-        //TODO disallow if there is an active game in the org this game belongs to
         Game game = await GetGameById(gameId);
 
+        if (game.Status != Game.GameStatus.New)
+        {
+            throw new ArgumentException($"Cannot start Game {gameId} because it has already been started");
+        }
+
+        Game newGame = await Collection.FindOneAndUpdateAsync<Game>(g => g.Id == gameId,
+            Builders<Game>.Update
+                .Set(g => g.Status, Game.GameStatus.Active)
+                .Set(g => g.StartedAt, _clock.GetCurrentInstant()),
+            new FindOneAndUpdateOptions<Game, Game>() { ReturnDocument = ReturnDocument.After }
+        );
+
+        await OnGameActiveStatusChanged(new(newGame, instigatorId, Game.GameStatus.Active));
+        return newGame;
+    }
+
+    public async Task<Game> SetGamePaused(string gameId, bool paused, string instigatorId)
```

Comment:
The reason that `StartGame` and `EndGame` are separate methods is because they are doing more than just setting the game status. Potentially pause/resume could be rolled into a `TogglePauseGame` method. Pause/resume has to get rolled together at some point, the question is where do we want to do that?

Reply:
the internal logic can still be the same no matter what method names we give it. But I'd like if the method signatures were all consistent for this functionality.